lib/nationbuilder/paginator.rb

```diff
@@ -11,10 +11,11 @@ def initialize(client, body)
       @body[page_type.to_s]
     end
 
-    define_method(:"#{page_type}") do
+    define_method(:"#{page_type}") do |args = {}|
       return nil unless send(:"#{page_type}?")
       path = send(:"#{page_type}?").split('/api/v1').last
-      results = @client.raw_call(path, :get)
+      args[:limit] ||= CGI.parse(path)['limit']
+      results = @client.raw_call(path, :get, args)
```

Comment:
To allow passing in of the `limit` param, it [defaulted](https://github.com/nationbuilder/nationbuilder-rb/blob/master/lib/nationbuilder/client.rb#L42) to an empty hash anyways when it wasn't included in the pagination before. Looking at this again, I'll update this to read `body` to be more in line with how the raw call is written.